README.md

```diff
@@ -41,6 +41,12 @@ git submodule update --init
 
 ### Source compile and install
 
+To install all required python packages run the following command:
+
+```
+python -m pip install -r requirements.txt
```

Comment:
Maybe use pip alias? `pip3 install -r requirements.txt`

Reply:
Or at least use `python -m pip install -r requirements.txt`, as above command will install packages for python 2, which I believe was not the intention.